src/main.cpp

```diff
@@ -157,7 +157,7 @@ int main(int argc, char** argv)
     new DirectoryResourceProvider("resources/");
     new DirectoryResourceProvider("scripts/");
     new DirectoryResourceProvider("packs/SolCommand/");
-    PackResourceProvider::addPackResourcesForDirectory("packs");
+    PackResourceProvider::addPackResourcesForDirectory("packs/");
```

Comment:
As pointed out in the SeriousProton pull request, I don't like the API change.

Reply:
I relaxed the API change.
However, I would argue this is worth keeping for the sake of consistency?
All other paths (besides mod) are currently forward slash terminated...

I'll revert this if it's really a bother though :)

Reply:
Oh, don't mind it here, only mind the API change :)